apps/typed-dependency-tester/README.md

```diff
@@ -0,0 +1,17 @@
+# Typed Dependency Tester
+
+The Typed Depndency Tester is a tool used to detect if types in `MicrosoftTeams.d.ts` are importing types from other modules and giving them an implicit `any` resolution
+
+## Getting Started
+
+### Running the Dependency Tester
+
+If you would like to use this tester locally, simply type `pnpm build` in your terminal. The tester will then proceed to install the workspace version of the TeamsJS.
```

Comment:
should be either `workspace version of TeamsJS` or `workspace version of the TeamsJS library`